pints/tests/test_log_likelihoods.py

```diff
@@ -559,30 +538,6 @@ def test_ar1(self):
         self.assertAlmostEqual(
             log_likelihood([0, 0.5, 5]), -19.706737485492436)
 
-        # multiple outputs
```

Comment:
Hiya! What's hapening here and below?

Reply:
Cool, sorry -- I didn't do the rebasing properly. See #752